pythia/cleaning/tests/test_elo.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import pandas as pd
+from pythia.cleaning import ELO
+from pathlib import Path
+from pythia.seo import Sunspotter
+from sunpy.util import SunpyUserWarning
+
+path = Path(__file__).resolve().parent.parent.parent.parent / "data/elo_test"
+
+
+@pytest.fixture
+def elo():
+    sunspotter = Sunspotter(timesfits=path / "../all_clear/lookup_timesfits.csv",
+                            properties=path / "../all_clear/lookup_properties.csv",
+                            classifications=path / "test_classifications.csv",
+                            classifications_columns=['image_id_0', 'image_id_1',
+                                                     'image0_more_complex_image1'])
+    column_map = {"player 0": "image_id_0",
+                  "player 1": "image_id_1",
+                  "score for player 0": "image0_more_complex_image1"}
+
+    return ELO(score_board=sunspotter.classifications, column_map=column_map)
+
+
+@pytest.mark.parametrize('rating_0,rating_1,expected_score',
+                         [(1400, 1400.0, 0.5),
+                          (1450, 1450.5, 0.49928044265518673),
+                          (1500, 1602.0, 0.3572869311673796),
+                          (1550, 1854.5, 0.14768898365874825),
+                          (1600, 2208.0, 0.029314241270450396)])
+def test_expected_score(elo, rating_0, rating_1, expected_score):
+    assert elo.expected_score(rating_0, rating_1) == expected_score
```

Comment:
Normally you wouldn't use `==` for testing that many decimals. Rather use `pytest.approx` or `np.testing.assert_allclose`